site/content/en/blog/releases/1.25.0-rc.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "1.25.0 - Release Candidate"
+linkTitle: "1.25.0-rc"
+date: "2022-07-26"
+---
+This is the 1.25.0 release of Agones.
+
+This release has lots of enhancements and bug fixes. This includes:
+
+* Move CustomFasSyncInterval to Beta.
+* Move NodeExternalDNS to stable.
+* Upgrade SuperTuxKart to 1.3.
+* Upgrade Xonotic to 0.8.5.
+* Add Alpha SDK support to the Unity SDK
```

Comment:
The other lines all end with a period. You should add a period here for consistency or take the other ones away.